src/settings/Settings.ts

```diff
@@ -134,6 +141,7 @@ export const SETTINGS: {[setting: string]: ISetting} = {
         ),
         supportedLevels: LEVELS_FEATURE,
         default: false,
+        controller: new IncompatibleController("feature_spaces"),
```

Comment:
Is this module missing? I don't see it locally...

Reply:
As per the PR body

> Based on https://github.com/matrix-org/matrix-react-sdk/pull/5668

Reply:
Ah huh, sorry, I guess I didn't notice / understand until now... 😅